packages/block-editor/src/components/link-control/style.scss

```diff
@@ -118,7 +118,7 @@ $preview-image-height: 140px;
 .block-editor-link-control__search-results {
 	margin-top: -$grid-unit-20;
 	padding: $grid-unit-10;
-	max-height: $grid-unit * 24;
+	max-height: $grid-unit * 24; // resolves to 194px
```

Comment:
I think the question I have is _why_ is it `194px`? Why did we choose that value? If it's just "that's the visual max height we want" or "it allows for 4 search results" then let's include that here as it will help future devs 🙏